src/screens/CollectionsSearchScreen.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
```

Comment:
Is this used anywhere? I see that it's imported into `AuthorizedNavigator` but I don't see any `navigation.navigate` triggers that go to `CollectionSearch`. If not, we should delete and remove the route in `AuthorizedNavigator`.

Reply:
removed this screen